src/lib/components/SplitContent.svelte

```diff
@@ -9,6 +9,13 @@
   import ContentBackdrop from "$lib/components/ContentBackdrop.svelte";
 
   export let back = false;
+  export const resetScrollPosition = () => {
+    if (scrollableElement) {
+      scrollableElement.scrollTop = 0;
+    }
+  };
+
+  let scrollableElement: HTMLElement | undefined;
```

Comment:
Is the `| undefined` necessary?

Reply:
Just for formality.  For example if we call `resetScrollPosition` on the next line, `scrollableElement` will be undefined.

Reply:
Have you tried that? I'm really not sure but I thought maybe because the `div` below is rendered unconditionally, this might always be defined.

Reply:
Yep, I tried.

Reply:
Thanks!